blockonomics.php

```diff
@@ -322,7 +311,8 @@ public function testSetup()
         $url = Configuration::get('BLOCKONOMICS_GET_CALLBACKS_URL');
         $response = $this->doCurlCall($url);
 
-        $callback_url = Configuration::get('BLOCKONOMICS_CALLBACK_URL');
+        $callbacksecret = Configuration::get('BLOCKONOMICS_CALLBACK_SECRET');
+        $callback_url = Tools::getHttpHost(true, true) . __PS_BASE_URI__ . 'modules/' . $this->name . '/callback.php?secret=' . $callbacksecret;
```

Comment:
This does work to change the callback URL, however, it will be better to fetch the base URL from the shop setting directly:
`Context::getContext()->shop->getBaseURL(true)`

Reply:
Done.